woocommerce/woocommerce-android
# [Dynamic Dashboard] Error view for when WCAdmin is disabled

Part of #11544 


### Description
This PR adds the following changes:
1. Introduces a common error view to use when WCAdmin is not available and the card's API depends on it.
2. Updates the Performance and Top Performers cards to use the new component.
3. Updates the Coupons card to use it too.

### Testing instructions
1. Disable WCAdmin (in a non-atomic website, WooCommerce -> Settings -> Advanced -> Features -> Enable WooCommerce Analytics)
2. Open the app.
3. Test Performance, Top Performers and Coupons cards.
4. Confirm all show the correct error view.

### Images/gif
<img src="https://github.com/woocommerce/woocommerce-android/assets/1657201/74229c32-301f-4c82-b93f-5130fbd3bd67" width=360 />
